### PR TITLE
Add defaults to the postgres exoscale XRD

### PR DESCRIPTION
**Pull request by ZuGao**
* Add defaults to the composition

## Checklist

- [x] PR contains a single logical change (to build a better changelog).
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.

### CODE REVIEW

**Comment by Kidswiss** [comment]
Okay I think didn't talk about the same thing 🗡️ 
My suggestion was to add it to the composite (XRD) not the composition, so it's visible via `kubectl explain`.

File: packages/composite/dbaas/exoscale/postgres.yml [hunk 1]
```diff
@@ -45,6 +45,7 @@ parameters:
                                   description: MajorVersion is the major version identifier for the
                                     instance.
                                   type: string
+                                  default: 14
```
Comment (Kidswiss) [inline packages/composite/dbaas/exoscale/postgres.yml:48]: You probably have to specify it as "14" becuase string :)

```suggestion
                                  default: "14"
```

**Review by Kidswiss — CHANGES_REQUESTED** [review]
Just one small thing, the rest LGTM.

Have you tested if it can still be applied? Or if there's some error somewhere?

**Comment by ZuGao** [comment]
Moved the defaults from composition to composite as suggested by @Kidswiss

**Review by Kidswiss — APPROVED** [review]

**Comment by ZuGao** [comment]
> Have you tested if it can still be applied? Or if there's some error somewhere?

I only test it with golden tests. And yes the output looks good.

**Review by ccremer — APPROVED** [review]

**Pull request merged**